package.json

```diff
@@ -11,8 +11,7 @@
     "docs": "yuidoc",
     "docs-deploy": "yuidoc && ghp-import -r origin -p -b gh-pages docs/",
     "start": "ember server",
-    "test": "ember test",
-    "check-style": "./node_modules/jscs/bin/jscs ."
+    "test": "./node_modules/jscs/bin/jscs . && ember test"
```

Comment:
These five steps are happening because we are forgetting to run things. This is only useful if  you are running npm test and forgetting to check the style. Adding it in is fine but please keep the check-style as a separate option without having to write it in full.